framework/src/st_things/things_stack/utils/things_ping.c

```diff
@@ -69,43 +74,191 @@ typedef struct things_ping_s {
+ * @param remote_addr : Remote address
+ * @param port : port number
+ * @return structure of things_ping_s
+ */
 static things_ping_s *create_things_ping_s(const char *remote_addr, const uint16_t port);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief State type prototype for terminate things_ping_s.
+ * @param ping : Structure of things_ping_s
+ */
 static void terminate_things_ping_s(things_ping_s *ping);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief Static type prototype for matching ip.
+ * @param ping : Structure of things_ping_s
+ * @param target_ip : IP of target device.
+ * @return success = 1, otherwise 0.
+ */
 static int is_ip_key_equal(things_ping_s *ping, const char *target_ip);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief Static type prototype for checking ping thread. Creating ping thread or
+ * destroy ping thread like that.
+ * @param ping : Structure of things_ping_s
+ */
 static void check_ping_thread(things_ping_s *ping);
+
+//--------------------------------------------------------------------------------
+/**
+ * @biref Static type prototype for set the state of Ping of things_ping_s.
+ * @param ping : Structure of things_ping_s
+ * @param state : State of Ping of things_ping_s
+ */
 static void set_mask(things_ping_s *ping, ping_state_e state);
+
+//--------------------------------------------------------------------------------
+/**
+ * @biref Static type prototype for get the state of Ping of things_ping_s.
+ * @param ping : Structure of things_ping_s
+ * @param state : State of Ping of things_ping_s
+ * @return success = true, failure = false.
+ */
 static bool get_mask(things_ping_s *ping, ping_state_e state);
+
+//--------------------------------------------------------------------------------
 static bool cas_mask(things_ping_s *ping, ping_state_e preState, bool preStateThen, ping_state_e post_state);
+
+//--------------------------------------------------------------------------------
+/**
+ * @biref Static type prototype for unset the state of Ping of things_ping_s.
+ * @param ping : Structure of things_ping_s
+ * @param state : State of Ping of things_ping_s
+ */
 static void unset_mask(things_ping_s *ping, ping_state_e state);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief Static type prototype for find ping resource to cloud.
+ * @param ping : Structure of things_ping_s
+ * @return 1 = TCP_session not connected, error case return sleeptime.
+ */
 static int find_resource_oic_ping(things_ping_s *ping);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief Static type prototype for sending common-Ping request for /oic/ping to Cloud.
+ * @parma ping : Structure of things_ping_s
+ * @return 1 = TCP_session not connected, error case return sleeptime.
```

Comment:
please check this return value descrpition. 
i think, in success case return sleeptime. 
error case return 0.

Reply:
fix.